src/classes/TelemetryService.cls

```diff
@@ -74,7 +74,7 @@ public without sharing class TelemetryService {
         private List<Telemetry> createCampaignTelemetries() {
             return new List<Telemetry>{
                 new Telemetry(Campaign.SObjectType, Telemetry.DeveloperName.CAMPAIGNS_WITH_JOBS)
-                    .withCriteria('Id IN (SELECT Campaign__c FROM Volunteer_Jobs)')
+                    .withCriteria('Id IN (SELECT Campaign__c FROM Volunteer_Job__c)')
```

Comment:
I'm concerned that neither of us caught this. Should we include a test that executes all telemetry queries just to confirm that they're valid queries?

Reply:
This was supposed to be that test but since the criteria was being dropped from the query builder it didn't actually include the criteria. I added a new test to the telemetry test to verify that the criteria is now being passed. When that intial fix was put in place this test started to fail. I should be using String.valueOf(SObject.FieldName) to build the query which prevents this type of thing from occurring but it makes the criteria a bit harder to read.